python/ray/tests/test_kill_raylet_signal_log.py

```diff
@@ -1,28 +1,51 @@
-import psutil
+import os
 import signal
 import sys
+import time
 import pytest
+import threading
+
 import ray
 
+from ray.test_utils import (
+    wait_for_pid_to_exit, )
+
+SIGKILL = signal.SIGKILL if sys.platform != "win32" else signal.SIGTERM
+
+
+def check_result(path):
+    with open(path) as f:
+        time.sleep(1)
+        s = f.read()
+        assert len(s) > 0
+        print(s)
+        assert "SIGTERM" in s
```

Comment:
For non-Windows environments, you send a SIGKILL to Raylet. Why do we always assert SIGTERM here?

Reply:
Windows no SIGABRT, but has SIGTERM, use SIGTERM to support windows and linux.

Reply:
For Linux and macOS, you send SIGKILL according to `signal.SIGKILL if sys.platform != "win32" else signal.SIGTERM`, right? But you assert `SIGTERM` here. So I suppose the test will fail on Linux and macOS.